Update DocumentPicker.getDocumentAsync API docs (#6652)
expo/expo: docs/pages/versions/unversioned/sdk/document-picker.md

#### Returns

> Returns a promise that resolves to an object as explained below.

Reviewer: ```suggestion
On success returns a promise that resolves to an object containing `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
If the user cancelled the document picking, the promise resolves to `{ type: 'cancel' }`.
```

> If the user cancelled the document picking, the promise returns `{ type: 'cancel' }`.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> Otherwise, it returns `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.

Reviewer: ```suggestion
```
